src/Service/NotificationReceiverService.php

```diff
@@ -90,6 +90,7 @@ public function __construct(
     public function process(Request $requestObject)
     {
         $request = $requestObject->request->all();
+        $salesChannelId = $requestObject->attributes->get('sw-sales-channel-id');
```

Comment:
If I'm not mistaken this is executed with no context, so there wouldn’t be a sales channel ID. Or is the Headless sales channel used?

Reply:
I believe the headless sales channel is used, I can get that the salesChannelId by inspecting the request parameterBag